packages/blockchain-link/src/workers/solana/index.ts

```diff
@@ -105,17 +105,35 @@ const pushTransaction = async (request: Request<MessageTypes.PushTransaction>) =
     const { lastValidBlockHeight } = await api.getLatestBlockhash('finalized');
 
     const txBuffer = Buffer.from(rawTx, 'hex');
-    const signature = await api.sendRawTransaction(txBuffer, {
-        skipPreflight: true,
-        maxRetries: 0,
-    });
 
-    await confirmTransactionWithResubmit(api, txBuffer, signature, lastValidBlockHeight);
+    try {
+        await api.sendRawTransaction(txBuffer, {
+            skipPreflight: true,
+            maxRetries: 0,
+        });
 
-    return {
-        type: RESPONSES.PUSH_TRANSACTION,
-        payload: signature,
-    } as const;
+        const signature = await api.sendRawTransaction(txBuffer, {
+            skipPreflight: true,
+            maxRetries: 0,
+        });
+
+        await confirmTransactionWithResubmit(api, txBuffer, signature, lastValidBlockHeight);
+
+        return {
+            type: RESPONSES.PUSH_TRANSACTION,
+            payload: signature,
+        } as const;
+    } catch (error) {
+        if (
+            error instanceof SendTransactionError &&
+            error.transactionError.message === 'Internal error'
```

Comment:
I'd probably add error`?`.transactionError`?`.message for greater runtime safety